README.md

```diff
@@ -137,6 +138,10 @@ View joint statistics about crash clusters:
 
     $ casr-cli casr_reports
 
+Convert reports to SARIF report:
+
+    $ casr-cli --sarif out.sarif --tool libfuzzer --source-root /xlnt casr/tests/casr_tests/casrep/test_clustering_san
```

Comment:
Can we provide a command example that actually can be executed from casr source directory? All other commands can be executed.